packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -835,7 +835,10 @@ export function signInActor({ services }: SignInMachineOptions) {
         },
         async getCurrentUserResolved(context) {
           groupLog('+++getCurrentUserResolved', context, event);
-          return await Auth.getCurrentUser();
+          return {
+            ...(await Auth.getCurrentUser()),
+            attributes: { ...(await Auth.fetchUserAttributes()) },
```

Comment:
Do we need to return the attributes for a specific reason?

Reply:
Not that I know of but a few of our example apps previously used values like `user.attributes.email` and previously user contained attributes in v5